main/common/settings.js

```diff
@@ -90,12 +92,16 @@ const audioInputDeviceId = store.get('audioInputDeviceId');
 (async () => {
   const devices = await audioDevices();
 
+  if (!Array.isArray(devices)) {
+    const Sentry = require('../utils/sentry');
+    Sentry.captureMessage(`devices is not an array: ${JSON.stringify(devices)}`);
```

Comment:
It's unclear from the [docs](https://github.com/getsentry/sentry-electron) if Sentry saves the message here when there's no exception. I think `captureMessage` is meant to add additional context to an error, but there's no error here. Let's use `captureException` just to be sure it's saved.

Reply:
// @karaggeorge  ⬆️

Reply:
Sorry, have been super busy at work lately 😅. Used `captureException`, their types say that it accepts `any` but the annotation says 

> an exception-like object

So, I passed in a new error. That should work right?

Reply:
Yup, I would assume so.